crates/top/rerun/src/commands/entrypoint.rs

```diff
@@ -57,12 +57,18 @@ Examples:
     Open an .rrd file and stream it to a Web Viewer:
         rerun recording.rrd --web-viewer
 
-    Host a Rerun gRPC server which listens for incoming connections from the logging SDK, buffer the log messages, and serves the results:
+    Host a Rerun gRPC server which listens for incoming connections from the logging SDK, buffers the log messages, and serves the results:
```

Comment:
Let's be consistent with the rest of the descriptions and use imperative:

```suggestion
    Host a Rerun gRPC server which listens for incoming connections from the logging SDK, buffer the log messages, and serve the results:
```